sql/base/dev/world_database.sql

```diff
@@ -992,17 +992,18 @@ CREATE TABLE `game_event_seasonal_questrelation` (
 /*!40101 SET character_set_client = @saved_cs_client */;
```

Comment:
Changes to this file should be reverted. world_database.sql only gets updated on TDB releases
(Travis starts by creating the world db, applies this file and only then all the SQL updates)